andela/ah-the-jedi-frontend
# #165305233 Users should be able to follow each other

#### What does this PR do?
   - Implements users can follow each other 

#### Description of Task to be completed?

    - implement functionality to follow a user
    - implement functionality to unfollow a user
    - ensure only logged in users can follow a user
    - get the usernames of followers and following
    - get the profiles of followers and following
    - render followers and following

### How should this be manually tested?

Clone this repository on local machine and CD into project folder
`$ git clone git@github.com:andela/ah-the-jedi-frontend.git`
`$ cd ah-the-jedi-frontend`

Check out to the PR's branch
`$ git checkout ft-user-follow-165305233`

install dependencies and run the dev server
`npm install`
`npm run start:dev`

navigate to profile page

### What are the relevant PT stories?
[#165305233](https://www.pivotaltracker.com/story/show/166571933)

### Screenshots
